### ISSUE TITLE
Add error variant and validation design to NativeSelect when it is a required field

### ISSUE THREAD

**Issue by BorghildSelle** [equinor/design-system#3691]
Hello,
we need to show validation error with NativeSelect, when NativeSelect is required field.
Could NativeSelect incorporate error variant and helpertext design as Textfield validation errors?

**Comment by oddvernes** [comment]
Yes the NativeSelect should probably have the same error variants as autocomplete

**Comment by millus** [comment]
The new (native)Select will now have error and helpermessage solved in #4918
